src/test/kotlin/tools/jackson/module/kotlin/test/github/failing/Github518.kt

```diff
@@ -41,7 +41,14 @@ class TestGithub518 {
      */
     @Test
     fun deserializeEmptyObjectToSingletonUnitWithSingletonSupport() {
-        val objectMapper = jsonMapper { addModule(kotlinModule { configure(SingletonSupport, true) }) }
+        val objectMapper = jsonMapper {
```

Comment:
I'd have preferred indentation not changed on this PR... but I guess it's what it is.